### PR TITLE
[NUI] Add some comments for NUI Layout

### PR DESCRIPTION
**Pull request by Seoyeon2Kim**
Signed-off-by: Seoyeon Kim <seoyeon2.kim@samsung.com>

### Description of Change ###
 - To avoid confusion of NUI Layout terms, add some summaries to `View` class.

### API Changes ###
- N/A

### CODE REVIEW

File: src/Tizen.NUI/src/public/BaseComponents/View.cs [hunk 1]
```diff
@@ -2393,6 +2396,8 @@ public Color Color
 
         /// <summary>
         /// Set the layout on this View. Replaces any existing Layout.
+        ///
+        /// Note : If a view explicitly sets `Layout` to null by user, it indicates that the view and its child views would not use Layout anymore.
```
Comment (dongsug-song) [inline src/Tizen.NUI/src/public/BaseComponents/View.cs:2400]: How about this?
<remarks> If this Layout is set as null explicitly, it means this View itself and it's child Views will not use Layout anymore. </remarks>

Comment (Seoyeon2Kim) [reply src/Tizen.NUI/src/public/BaseComponents/View.cs:2400]: Good good! Thanks for the review!! :+1:

**Review by rabbitfor — APPROVED** [review]
LGTM!

**Review by tscholb — APPROVED** [review]
LGTM!

**Review by jaehyun0cho — APPROVED** [review]

**Review by huiyueun — APPROVED** [review]

**Pull request merged**